lottie/src/main/java/com/airbnb/lottie/manager/ImageAssetManager.java

```diff
@@ -102,6 +102,25 @@ public void setDelegate(@Nullable ImageAssetDelegate assetDelegate) {
       return putBitmap(id, bitmap);
     }
 
+    //if no images folder given assume drawable res
+    if (TextUtils.isEmpty(imagesFolder)) {
```

Comment:
@sirpy Can you add some snapshot tests for this in `LottieTests`?

Reply:
good question:)

Reply:
https://github.com/airbnb/lottie-android/blob/master/LottieSample/src/androidTest/java/com/airbnb/lottie/samples/LottieTest.kt

Reply:
i'm afraid I don't really understand the testing structure and where the test files are,
dont have the resources to dig into it at the moment, would appreciate help on that

Reply:
@gpeal

Reply:
@gpeal